Bottom: cfb25c0514733885e27dcb031bfb6d12abc8f4f4
Top:    9b39ae675fb3c1dc0bf13adc6e13fa3a0962584a
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-24 10:39:03 +0800

Refresh of vhost-introduce-helper-to

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 130bebc..4b92e33 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2456,6 +2456,7 @@ struct vhost_msg_node *vhost_dequeue_msg(struct vhost_dev *dev,
 }
 EXPORT_SYMBOL_GPL(vhost_dequeue_msg);
 
+
 static int __init vhost_init(void)
 {
 	return 0;
